.neuro/live-s3.yml

```diff
@@ -227,9 +177,8 @@ jobs:
       - ${{ params.mlflow_storage }}/0/${{ params.run_id }}/artifacts/model/data/model.h5:/storage/model.h5
     env:
       PYTHONPATH: /usr/project
-      PROJECT: /usr/project
       EXPOSE_SSH: "yes"
-      DATA_PATH: ${{env.PROJECT}}/data/Images
+      DATA_PATH: /usr/project/data/Images
```

Comment:
Why not? One can run Jupyter for some other needs, such as running commands to populate data or just testing something

Reply:
yes, you are correct. Let's put only `T` into `neuro blob cp -rT blob:${BUCKET_NAME}/images ${DATA_PATH}`, since currently the path in the notebook is broken.

Reply:
Ok, done!